core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -2432,11 +2432,6 @@ public String getUrlChildPrefix() {
      */
     public @Nullable String getRootUrl() throws IllegalStateException {
         final JenkinsLocationConfiguration config = JenkinsLocationConfiguration.get();
-        if (config == null) {
-            // Try to get standard message if possible
-            final Jenkins j = Jenkins.get();
-            throw new IllegalStateException("Jenkins instance " + j + " has been successfully initialized, but JenkinsLocationConfiguration is undefined.");
-        }
```

Comment:
This can never be null because `ExtensionList#getInstance` throws `IllegalStateException` if the class cannot be found.